src/main/resources/view/DarkTheme.css

```diff
@@ -218,7 +218,7 @@
     -fx-border-width: 2;
     -fx-background-radius: 0;
     -fx-background-color: #1d1d1d;
-    -fx-font-family: "Segoe UI", Helvetica, Arial, sans-serif;
+    -fx-font-family: "Open Sans Regular", Helvetica, Arial, sans-serif;
```

Comment:
Given that our preferred font is always available, is it still necessary for the fallback fonts to be around?

Another thought I have is that could the better fix be including the fallback fonts for every element? Of course it might be a little late at this stage to change the direction of the fix, but I'm just throwing this out for everyone to chime in.

Reply:
Since we are distributing the preferred font I think we can remove the fallback fonts. In any case Helvetica and Arial may be unavailable on some systems, and including a fallback to sans-serif may lead to any inconsistencies or bugs being unnoticed and unreported.

Reply:
Seems like this stems from the initial commit where the css was borrowed from a guide, so there should not be an issue removing the fallback fonts!

Reply:
I agree, since the fallback fonts would not be used they can be removed. Will also fix the indentation issue as well. Thanks @kouyk for the input.